app/webpacker/src/stylesheets/_feedback.scss

```diff
@@ -18,3 +18,7 @@
     color: red;
   }
 }
+
+#feedback-message-field-error { 
```

Comment:
Why do we need to manually style the error on the form, this isn't particularly scalable since we'd need to fix this for every form? The form builder be handling this already, if its not it probably means something else is up?

Reply:
Looks like it _should_ just be working?

https://govuk-form-builder.netlify.app/introduction/error-handling/

Reply:
Looks like an upstream bug - js is overwriting the class I think